services/event/config/config.go

```diff
@@ -62,5 +64,11 @@ func Initialize() error {
 
 	EVENT_CHECKIN_TIME_RESTRICTED = (checkin_time_res_str == "true")
 
+	TOKEN_SECRET, err = cfg_loader.Get("TOKEN_SECRET")
+
+	if err != nil {
+		return err
+	}
+
```

Comment:
Since `TOKEN_SECRET` exists in common now, we shouldn't need to define it like this for every file, and instead just use common_config.

Reply:
Nice catch. Yeah i set those up prior to making the decision to put `TOKEN_SECRET` in `common`.